src/runtime/metal/metal_module.mm

```diff
@@ -71,6 +73,28 @@ void SaveToBinary(dmlc::Stream* stream) final {
       return "";
     }
   }
+
+  std::unordered_map<std::string, std::string> SplitKernels(std::string source) const {
+    std::unordered_map<std::string, std::string> split_kernels;
```

Comment:
This is a duplicated util from https://github.com/apache/tvm/blob/main/src/runtime/opencl/opencl_module.cc#L245. consider move to a common one. e.g. src/runtime/source_utils.h

Reply:
Done. Thank you.